api/client/proto/authservice.proto

```diff
@@ -849,15 +811,15 @@ service AuthService {
         returns (types.ResetPasswordTokenV3);
 
     // GetUser gets a user resource by name.
-    rpc GetUser(GetUserRequest) returns (types.UserV2);
+    rpc GetUser(types.GetResourceWithSecretsRequest) returns (types.UserV2);
```

Comment:
Are all the modified RPCs here new since 6.0?
It's not safe to change RPC message types when there are clients/server out there using the previous version